### DISCUSSION TITLE
r/china-travel-layoff-trip-day7-yixian-daguling

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#79] [Announcements]
# r/china-travel-layoff-trip-day7-yixian-daguling

超推荐的一个小众玩儿水的地方，有瀑布、有水潭，清澈干净，趣味盎然

https://blog.ops-coffee.cn/r/china-travel-layoff-trip-day7-yixian-daguling

**Comment by xinzhang-hello** [comment]
带娃的好去处！
